ts/components/messages/MessageListComponent.tsx

```diff
@@ -12,7 +12,8 @@ import { MessageState } from "../../store/reducers/entities/messages/messagesByI
 import { PaymentByRptIdState } from "../../store/reducers/entities/payments";
 import { ServicesByIdState } from "../../store/reducers/entities/services/servicesById";
 import { MessageListItemComponent } from "./MessageListItemComponent";
-
+// tslint:disable-next-line: no-var-keyword
+var messagesToRead = 0;
```

Comment:
don't use vars, the tslint rule is there for a reason :)

Reply:
Ok, but if i don't use vars what kind of variable should I use to update badge?